Test: issue #12 add  test cases for SetSizeLimits

EdwardNorberg opened the pull request:
add 5 test cases for SetSizeLimit function in
internal/goglfw/window_windows.go

EdwardNorberg committed:
Test: issue #12 add  test cases for SetSizeLimit

add 5 test cases for SetSizeLimit function in
internal/goglfw/window_windows.go

erikcghedlund reviewed (commented):
This will result in a single test case and will only show failure on the first fail. Consider breaking the function up into many functions with one atomic test case

EdwardNorberg committed:
Refactor: issue #12 split tests into several funcs

split up tests for SetSizeLimits into several functions.
Tests ran locally and still work
